.github/workflows/ci.yml

```diff
@@ -272,6 +272,11 @@ jobs:
             sudo udevadm control --reload-rules
             sudo udevadm trigger --name-match=kvm
     steps:
+    # Github runners seem to have intermittent connectivity issues.
+    # See https://github.com/beeware/toga/issues/2632
+    - name: Tune GitHub-hosted runner network
+      uses: smorimoto/tune-github-hosted-runner-network@v1
```

Comment:
This user providing this Action doesn't seem like they are likely to turn nefarious at some point...but pinning just `v1` means we'll automatically start running any changes they merge. That said, the README says they plan to make no changes, so even pinning it to a specific commit hash should be fine. Throwing this out there as, at least, an observation since this is all pretty small risk.

Reply:
Fair; I'll update the pin to 1.0.0.